std/node/process_test.ts

```diff
@@ -1,37 +1,11 @@
 // deno-lint-ignore-file no-undef
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 
+import "./global.ts";
 import { assert, assertEquals, assertThrows } from "../testing/asserts.ts";
+import { stripColor } from "../fmt/colors.ts";
 import * as path from "../path/mod.ts";
-import * as all from "./process.ts";
-import { argv, env } from "./process.ts";
 import { delay } from "../async/delay.ts";
-import "./global.ts";
-
-// NOTE: Deno.execPath() (and thus process.argv) currently requires --allow-env
-// (Also Deno.env.toObject() (and process.env) requires --allow-env but it's more obvious)
-
-Deno.test({
-  name: "process exports are as they should be",
```

Comment:
Why remove this test?

Reply:
Previously this test was under the assumption the library would only expose things available through the ESM export (for some reason they are different to the items exposed through require), however since most Node packages used in Deno are transforms from CommonJS and modules used through createRequire not exposing this properties makes no sense

This test was used to check that the export from process was indeed different to the built in, but this is no longer the case